cpp/CMakeLists.txt

```diff
@@ -68,7 +68,7 @@ set(CUSPATIAL_BUILD_TESTS ${BUILD_TESTS})
 set(CUSPATIAL_BUILD_BENCHMARKS ${BUILD_BENCHMARKS})
 
 set(CUSPATIAL_CXX_FLAGS "")
-set(CUSPATIAL_CUDA_FLAGS --expt-extended-lambda --expt-relaxed-constexpr)
```

Comment:
Do we also want to remove `--expt-extended-lambda` here? I think it's a good idea to remove exception flags, just to make sure this is intended since it's not mentioned in the PR desc.

Reply:
I think it's fine to remove here since it's set in `ConfigureCUDA.cmake`. This was redundant.